src/models/Super-admin.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const superAdminsSchema = new Schema({
+  email: {
+    type: String,
+    required: true,
+  },
+  password: {
+    type: String,
+    match: /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]{8,}$/,
+    required: true,
+  },
+});
+
+module.exports = mongoose.model('Super-admin', superAdminsSchema);
```

Comment:
- the correct param name in mongoose model is 'Super-Admin'
- Rename the file using PascalCase